deploy/cr.yaml

```diff
@@ -31,7 +31,12 @@ spec:
 #      iam.amazonaws.com/role: role-arn
 #    labels:
 #      rack: rack-22
-#   schedulerName: "default"
```

Comment:
I think `schedulerName` still exists

Reply:
there were two schedulerNames in one section. It's just a cleanup